### ISSUE TITLE
Export formated SQL with "IN" in where-clause may 

### ISSUE THREAD

**Issue by Tanarri** [joyfullservice/msaccess-vcs-addin#486]
Hi,

I found a reproduceable issue at exporting SQL-Query, when in the settings "format SQL" is checked.
The affected queryies (6 queries in total) in my project relatively complex, but after a few tests I was able to reproduce the error with this very simple query.

The SQL Query, which is produce a faulty formatted SQL is:
(yes, I know that the SQL command makes no sense)
```
SELECT MSysQueries.Attribute, MSysQueries.Flag
FROM MSysQueries
WHERE (((MSysQueries.Flag) In (SELECT Flag from [MSysQueries] where flag=0)));
```

which will be exported faulty as
```
SELECT
   MSysQueries.Attribute,
   MSysQueries.Flag
 FROM
   MSysQueries
 WHERE
   (
     (
       (MSysQueries.Flag) In (
         SELECT
           Flag
         from
           [MSysQueries]
         where
           flag = 0
         )
       )
     );
 WARNING: unclosed parentheses or section
```

An now the slightly modified SQL, which will be exported correctly:
```
SELECT MSysQueries.Attribute, MSysQueries.Flag
FROM MSysQueries
WHERE (((MSysQueries.Flag) In (SELECT Flag from [MSysQueries])));
```

which will be exported correct as
```
SELECT
  MSysQueries.Attribute,
  MSysQueries.Flag
FROM
  MSysQueries
WHERE
  (
    (
      (MSysQueries.Flag) In (
      SELECT
        Flag
      from
        [MSysQueries])
    )
  );
```

Please note that the problem raises when a "WHERE" clause in the sub-select is used.

Hope I could gave you the need information for find and fixing the issue

Greetings

**Comment by joyfullservice** [comment]
I think I was able to successfully resolve this issue... Would you be able to do some testing across your systems to make sure it didn't introduce any unintended side effects? With this change I am deviating slightly from the original logic this was based upon, but I think it does improve it by adding support for some more complex SQL queries.

**Comment by Tanarri** [comment]
I tested your patch on my access database and all worked smooth on export.
In my backend VCS (SVN) I saw, that only the problematic queries got changed.
Additional 2 others queries got formatted on export, which weren't formatted on export with the last version of VCS. But there are ok.

I rebuild my access database from source and all queries are now imported as the should.

One special query has in its formatted SQL a little issue, but the query work! It's only a cosmetic issue and I will report it in a seperate ticket.

@joyfullservice : You are great! Thank you!
